lib/jobs/create-default-pollers.js

```diff
@@ -98,22 +98,22 @@ function createDefaultPollersJobFactory(
                 delete poller.type;
                 return self.createRedfishPoller(nodes, poller);
             } else {
-                var sourceValue;
+                var sourceQuery;
                 if (poller.type === 'ipmi') {
                     poller.name = Constants.WorkItems.Pollers.IPMI;
                     delete poller.type;
-                    sourceValue = 'bmc';
+                    sourceQuery = {'$in':[/^bmc/, 'rmm']};
```

Comment:
I have concern about using raw mongodb query, we'd better to use waterline syntax as we are planing to support other databases.